spec/rspec/rails/example/controller_example_group_spec.rb

```diff
@@ -19,6 +19,23 @@ def group_for(klass)
       expect(group.included_modules).to include(RSpec::Rails::Matchers::RoutingMatchers)
     end
 
+    it "handles methods invoked via `method_missing` that use keywords" do
+      group =
+        RSpec::Core::ExampleGroup.describe ApplicationController do
+          def a_method(value:); value; end
+          def method_missing(_name, *_args, **kwargs, &_block); a_method(**kwargs); end
+
+          # This example requires prepend so that the `method_missing` definition
+          # from `ControllerExampleGroup` bubbles up to our artificial one, in reality
+          # this is likely to be either an internal RSpec dispatch or one from a 3rd 
+          # party gem.
+          prepend ControllerExampleGroup
```

Comment:
Is this statement purposefully put to the end of the block?
Moving it above own's `method_missing` definition makes no difference, and includes/prepends are commonly at the beginning of the class definition.

Reply:
This was the implementation of a failing spec I was happy with